docs/user/basics/documents-creator.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Documents Creator
+description: Create user-friendly documents
+lang: en
+---
+
+# Documents Creator
+
+The Documents creator converts your documentation data into user-friendly documents. This allows you to reach a diverse audience - from tech-savvy stakeholders to the less digitally fluent decision-makers.
+
+The documents can serve as a documentation basis for audits, conformity checks and customer reporting. Documents are available in the main navigation bar.
+
+## Rights
+
+You should check the [Rights](../../admin/rights-and-permissions.md) and alter them to fit your needs. It is also possible to set `All available Rights (inlcuding future updates)`.
+
+## Design your documents
+
+One size doesn't fit all, especially in the world of IT Asset Management. Different stakeholders have different needs. Customize your document with different chapters and sections and decide what elements of your IT asset data take center stage.
+
+### Global documents
+
+Global documents can be viewed by any tenant.
+
+### Tenant documents
+
+Tenant documents are only available for the current tenant.
+
+### Document settings
+
+Each document needs to have a Document title and can have a description.
+
+### Chapters
+
+There are default Chapter `Title page` and `Table of content`. Also there is a add button to add `custom chapter` for your needs.
+
+#### Sections
+
+To fill the Chapter with data you have to Option to add `Object details` section, `Text` section, `Headline` section and a `Custom Table`. The `Headline` is added by default. In the `Text` section you can use supported [Markdown](https://www.markdownguide.org/basic-syntax/) to build high quality documents. Sections can be reordered or deleted.
+
+!!! tip "Text editor"
+    If you click on the `Text editor` button you can easily edit your text in the `Text editor` popup with a Toolbar and preview.
+
+In the `Object details` section the attributes are displayed per row.
+The `Custom Table` section displays the result like a table.´
+
+And you define conditions to narrow down the result set, by editing the `Objects shown in section`. Also you define the displayed `Attributes`.
+
+### Sending documents to the target group
+
+Accelerate your communication: Instead of manually crafting and distributing documents, you can use automated mailings to swiftly and regularly compile the latest and most relevant information and send it to your audience.
+
+### Mailings
+
+Each mailing list needs a title, recipient(s) which can also be added by direct input, a subject and a message.
+After that you can set a execution date and configure if the mailing should be done in an interval.
+When you check `Repeat execution` you can select a `End date of execution` and a interval.
```

Comment:
you can select a `End date of execution` and a interval
->
you can select `End date of execution` and an interval